docs/jwst/extract_1d/description.rst

```diff
@@ -239,25 +247,27 @@ direction perpendicular to dispersion.
 
 Extraction for 3D IFU Data
 --------------------------
-For IFU cube data, 1D extraction is contolled by a different set of EXTRACT1D
-reference file parameters.
-Note that for an extended source, anything specified in the reference file
+For IFU cube data, 1D extraction is controlled by a different set of EXTRACT1D
```

Comment:
This phrase seems to end unexpectedly and doesn't flow to the wording in the next line